sw/host/ot_certs/src/template/mod.rs

```diff
@@ -86,6 +89,13 @@ pub struct BasicConstraints {
     pub ca: Value<bool>,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct SubjectAltName {
```

Comment:
Is this structure used?

Reply:
I guess not, a leftover from the original implementation, dropped.